dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -473,10 +473,8 @@ def _exclusion(providers: Iterable[str]) -> str:
 BASE_PROVIDERS_COMPATIBILITY_CHECKS: list[dict[str, str]] = [
     {
         "python-version": "3.8",
-        "airflow-version": "2.6.0",
-        "remove-providers": _exclusion(
-            ["openlineage", "common.io", "cohere", "fab", "qdrant", "microsoft.azure"]
-        ),
+        "airflow-version": "2.7.0",
```

Comment:
You should just remove 2.7.0 -> there is a small incompatibility in the cohere provider with importlib, that makes it technically `2.7.1` compatible not 2.7.0 - that's why 2.7.1 is added below. I think it's not worth to complicate things and exclude `cohere` for 2.7.0 additionally - technically there is a very small differerence between 2.7.0 and 2.7.1 so I'd rather leave 2.7.1.

Reply:
https://github.com/apache/airflow/pull/39240/commits/c3d85aa845bb854f065037035ef6d84e4d7fee02